### PR TITLE
Add Grand Winner Divyam Tayal

### PR DESCRIPTION
**Pull request by daretobedifferent18**
Issue #295 
Screenshot from 2021-01-01 18-39-10

### CODE REVIEW

**Comment by mariobehling** [comment]
Please provide a color headshot image instead of a very dark image.

**Review by Raunaksingh100 — COMMENTED** [review]
It is very great you can add all the winners and Finalist of this 1 coding contents in this pr...

You will get images from the GitHub profile.

**Comment by daretobedifferent18** [comment]
@mariobehling Should I add all the winners or just mine?

**Comment by Raunaksingh100** [comment]
> @mariobehling Should I add all the winners or just mine?

You can add all the winners and grand prize winner see in the issue there is need of all not the one..

**Comment by mariobehling** [comment]
Each winner can make their own PR. You only need to resolve merge conflicts later if they appear, but that should not be too difficult for you guys.

File: index.html [hunk 1]
```diff
@@ -693,6 +693,42 @@ <h1 class="text-white">About "Codeheat"</h1>
...
+		<section class="speakers duplicatable-content">
+
+			<div class="container">
+				<div class="row">
+					<div class="col-sm-12">
+						<h1>Grand Winners</h1>
+					</div>
+				</div>
+
+				<div class="row">
```
Comment (sachinchauhan2889) [inline index.html:706]: closing tag if this div i.e </div> not present.

File: index.html [hunk 2]
```diff
@@ -693,6 +693,42 @@ <h1 class="text-white">About "Codeheat"</h1>
 			</div>
 		</section>
 
+		<a id="winners" class="in-page-link"></a>
+		<section class="speakers duplicatable-content">
+
+			<div class="container">
```
Comment (sachinchauhan2889) [inline index.html:699]: closing tag if this div i.e not present.

File: index.html [hunk 3]
```diff
@@ -693,6 +693,42 @@ <h1 class="text-white">About "Codeheat"</h1>
...
+										</a>
+									</div>
+								</div>
+							</div>
+							<span class="speaker-name">DIVYAM TAYAL</span>
+							<span>Developer at FOSSASIA</span>
+						</div>
+					</div>
+	<!--	</div>-->
+	</section>
```
Comment (sachinchauhan2889) [inline index.html:731]: have wrong identation.

File: index.html [hunk 4]
```diff
@@ -693,6 +693,42 @@ <h1 class="text-white">About "Codeheat"</h1>
 			</div>
 		</section>
 
+		<a id="winners" class="in-page-link"></a>
+		<section class="speakers duplicatable-content">
+
+			<div class="container">
+				<div class="row">
+					<div class="col-sm-12">
+						<h1>Grand Winners</h1>
```
Comment (sachinchauhan2889) [inline index.html:702]: Grand Prize Winnners

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
Divyam i have copied your code from this PR. while editing your code into my pr I found these. you can apply these changes to your PR also.

**Comment by sachinchauhan2889** [comment]
you can also remove left padding of h1 containing "grand prize winning" because picture and text are are perfectly aligned.

**Comment by daretobedifferent18** [comment]
@mariobehling I don't have any other pic right now. Could you accept this one only

**Comment by iamareebjamal** [comment]
@daretobedifferent18 Please update

**Comment by daretobedifferent18** [comment]
@iamareebjamal Done

**Pull request merged**